Project-1/README.md

```diff
@@ -13,111 +13,345 @@ Note: The standard interpretation of the logical symbols - "∨", "∧", "→",
+  
 ```
 
 5. Select two formulas defined in ALC from question 4 to form the basis of a T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both. 
+```
+Tex = {Brother = M∩∃parent_of¯.(≥2∃parent_of)			(Tex.1)
+	Aunt = ~M∩∃parent_of¯.(∃parent_of.(∃parent_of))		(Tex.2)
+	Brother ⊆ Male,						(Tex.3)
+	Aunt ⊆ ~Male}						(Tex.4)
+
+Aex = {Billy: Brother ⊓ Male		(Aex.1)
+	Bob: Brother ⊓ Male		(Aex.2)
+	Mary: Aunt			(Aex.3)
+	(Alex, Billy): parent_of	(Aex.4)
+	(Alex, Bob): parent_of}		(Aex.5)
+	
+ΔI = {w, x, y, z},
+Billy = w
+Bob = x
+Mary = y
+Alex = z
+Male = {w, x}
+Brother = {w, x}
+Aunt = {y}
+Parent_of = {(z,w), (z,x)}
+```
 
 6. Explain the difference - using natural language - between the first-order prefixes:
-  ```(a) ∃x∀y and ∀x∃y
+  ```
+  (a) ∃x∀y and ∀x∃y
+  
+  ∃x∀y means "there is an x such that, for all y...", while ∀x∃y means "for all x, there is a y such that...". For instance, if I wanted to say that there is one particular thing that everything loves, I would write: "∃x∀yL(y,x)", but if I wanted to say that everything loves something without implying they all love the same thing, I would write: "∀x∃yL(x,y)". 
+  
   (b) ∃x∀y∃z and ∀x∃y∀z 
+  
+  ∃x∀y∃z means "there is an x such that, for all y, there is a z such that..." while ∀x∃y∀z means "for all x, there is a y such that, for all z..."
+  
   (c) ∀x∃y∀z∃w and ∃x∀y∃z∀w
+  
+  ∀x∃y∀z∃w means "for all x, there is a y such that, for all z, there is a w such that...", while ∃x∀y∃z∀w means "there is an x such that, for all y, there is a z such that, for all w..."
+  
 ```
 	
 7. Show that the following sentences are not equivalent by exhibiting a graph that models one but not both of these sentences:
 ```
 ∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
 ∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+
+
+	0------> 1
+       ^ |      | ^
+       |_|      |_|
+       
+       -I'm not sure how to draw a graph in this program. I'm trying to show object 0 being in a relation with itself and with 1, but 1 only being in the relation with itself.
+       U: {0, 1}
+       R: {(0,0), (0,1), (1,1)}
+	-This fits with the second statement, since there is an object that Rs everything (0), there is an object that Rs something (both 0 and 1), and it's true that everything Rs something (both 0 and 1 do, and they're the only objects)
+	-This does not fit with the first statement, since it's not true that everything Rs everything "R(x,z)" (1 does not).
+       
 ```
 	
 8. Using an online tableau proof generator - such as the one found here `https://www.umsu.de/trees/` - provide tree proofs of the following entailments, which are known as the De Morgan's laws:
-  ```(a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+  ```
+  (a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+  
+  	1.¬∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+ 	2.¬∀y(¬(Pa ∧ Qa) → (¬Pa ∨ ¬Qa))(1)
```

Comment:
What justifies moving from a negated universal in line 1 to a negated universal in line 2?